projects/igniteui-angular/src/lib/grids/pivot-grid/pivot-grid-navigation.service.ts

```diff
@@ -67,6 +72,35 @@ export class IgxPivotGridNavigationService extends IgxGridNavigationService {
         }
     }
 
+    public override handleAlt(key: string, event: KeyboardEvent): void {
+        event.preventDefault();
+
+        let row = this.grid.gridAPI.get_row_by_index(this.activeNode.row);
+        let expansionRowKey = PivotUtil.getRecordKey(row.data, row.data.dimensions[0])
+        let isExpanded = this.grid.expansionStates.has(expansionRowKey) ? this.grid.expansionStates.get(expansionRowKey) : true;
+
+        if (!isExpanded && ROW_EXPAND_KEYS.has(key)) {
+            if (row.key === undefined) {
```

Comment:
Is there instances where there's no key for the row? I think that might be true for the base grid, but the pivot always generates keys so we probably will never have such a scenario. So this can be simplified.